### PR TITLE
`repo-header-info` - Bugfix: Also appears in wrong place

### LINKED ISSUE #6923: `repo-header-info` shows info for ci actions
### Description

Repo-header-info show the stars and lock on to many elements. On the CI actions popover I hav enoticed this.

image

### How to replicate the issue + URL

1. Goto https://github.com/refined-github/refined-github
2. Click on the CI checkmark

### Extension version

23.9.17

### Browser(s) used

Chrome

### PR DESCRIPTION
**Pull request by 134130**
## Description
- Closes #6923

## Test URLs
- https://github.com/refined-github/refined-github
- Click on the CI checkmark

## Screenshot

- Before
  image

- After
  image

### CODE REVIEW

File: source/features/repo-header-info.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ async function add(repoLink: HTMLAnchorElement): Promise<void> {
 }
 
 function init(signal: AbortSignal): void {
-	observe('header .AppHeader-context-full li:last-child a', add, {signal});
+	observe('header .AppHeader-context-full li:last-child a:has(span.AppHeader-context-item-label)', add, {signal});
```
Comment (134130) [inline source/features/repo-header-info.tsx:58]: Do safari still not supports `:has()` selector? If it is, we can use below

```suggestion
	observe('header .AppHeader-context-full li:last-child > a', add, {signal});
```

Comment (FloEdelmann) [reply source/features/repo-header-info.tsx:58]: Why not like this?

```suggestion
	observe('header .AppHeader-context-full li:last-child a.AppHeader-context-item', add, {signal});
```

Firefox does not support `:has` yet by default, see #5797. Note that the `> a` selector would probably break #6926.

Comment (134130) [reply source/features/repo-header-info.tsx]: @FloEdelmann Thank you! I missed that..

**Comment by fregante** [comment]
Thank you for the fixes!

**Comment by 134130** [comment]
Do not merge yet please. It shoud be tested on old UI.

**Review by fregante — APPROVED** [review]

**Pull request merged**